FF1Lib/Assembly/BA.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FF1Lib.Assembly
+{
+	public enum MemoryMode
+	{
+		MMC1 = 1,
+		MMC3 = 3
+	};
+
+	/// <summary>
+	/// Bank and (run) Address.
+	/// </summary>
+	public struct BA
```

Comment:
I'd like a longer name for this, but I'm struggling to come up with a good one (the two hardest things in programming are naming things and cache invalidation).  `BankAddress` is probably fine.